app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,15 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <%= link_to(image_tag("3sixd.png", :class => "navbar-brand", :alt => "Company logo"), root_path) %>
+      <% if Company.last.logo.present? %>
```

Comment:
@leonardk1 Its OK, but i did put some comments about this `Company.first` and `Company.last` in the first PR, describing that they will be removed when I implement the authentication